balancer-js/README.md

```diff
@@ -228,19 +228,17 @@ const pricing = new Pricing(sdkConfig);
 
 ### #getSpotPrice
 
-Calculates Spot Price for a token pair - for specific pool if ID otherwise finds most liquid path and uses this as reference SP.
+Calculates Spot Price for a token pair - finds most liquid path and uses this as reference SP.
```

Comment:
Should we have calcSpotPrice on readme as well? Or getSpotPrice is already sufficient?